.gitignore

```diff
@@ -16,3 +16,6 @@
 
 # Ignore bin directory contents
```

Comment:
Prob can just copy this over?
https://github.com/kcp-dev/kcp-glbc/blob/main/.gitignore